plugins/woocommerce/src/Admin/API/AI/StoreTitle.php

```diff
@@ -40,63 +30,58 @@ class StoreTitle extends AbstractRoute {
 	const DEFAULT_TITLE = 'Site Title';
 
 	/**
-	 * Get the path of this REST route.
+	 * Endpoint namespace.
 	 *
-	 * @return string
+	 * @var string
 	 */
-	public function get_path() {
-		return self::get_path_regex();
-	}
+	protected $namespace = 'wc-admin';
 
 	/**
-	 * Get the path of this rest route.
+	 * Route base.
 	 *
-	 * @return string
+	 * @var string
 	 */
-	public static function get_path_regex() {
-		return '/ai/store-title';
-	}
+	protected $rest_base = 'ai';
 
 	/**
-	 * Get method arguments for this REST route.
-	 *
-	 * @return array An array of endpoints.
+	 * Register routes.
 	 */
-	public function get_args() {
-		return array(
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/store-title',
 			array(
-				'methods'             => \WP_REST_Server::CREATABLE,
-				'callback'            => array( $this, 'get_response' ),
-				'permission_callback' => array( Middleware::class, 'is_authorized' ),
-				'args'                => array(
-					'business_description' => array(
-						'description' => __( 'The business description for a given store.', 'woocommerce' ),
-						'type'        => 'string',
+				array(
+					'methods'             => \WP_REST_Server::CREATABLE,
+					'callback'            => array( $this, 'update_store_title' ),
+					'permission_callback' => array( Middleware::class, 'is_authorized' ),
+					'args'                => array(
+						'business_description' => array(
+							'description' => __( 'The business description for a given store.', 'woocommerce' ),
+							'type'        => 'string',
+						),
 					),
 				),
-			),
-			'schema'      => array( $this->schema, 'get_public_item_schema' ),
-			'allow_batch' => array( 'v1' => true ),
+				'schema' => array( $this, 'get_schema' ),
+			)
 		);
 	}
 
 	/**
 	 * Update the store title powered by AI.
 	 *
-	 * @param  \WP_REST_Request $request Request object.
+	 * @param  WP_REST_Request $request Request object.
 	 *
-	 * @return bool|string|\WP_Error|\WP_REST_Response
+	 * @return WP_Error|WP_REST_Response
 	 */
-	protected function get_route_post_response( \WP_REST_Request $request ) {
+	public function update_store_title( $request ) {
 
 		$business_description = $request->get_param( 'business_description' );
 
 		if ( ! $business_description ) {
-			return $this->error_to_response(
-				new \WP_Error(
-					'invalid_business_description',
-					__( 'Invalid business description.', 'woocommerce' )
-				)
+			return new WP_Error(
+				'invalid_business_description',
+				__( 'Invalid business description.', 'woocommerce' )
 			);
 		}
 
```

Comment:
Can we update `$store_title = html_entity_decode( get_option( 'blogname' ) );` to use the `STORE_TITLE_OPTION_NAME` const